eth-repository/src/main/kotlin/org/apache/tuweni/eth/repository/BlockchainIndex.kt

```diff
@@ -553,16 +550,16 @@ class BlockchainIndex(private val indexWriter: IndexWriter) : BlockchainIndexWri
 
       val topDocs = searcher!!.search(
         TermQuery(Term("_type", "difficulty")),
-        10,
+        HITS,
         Sort(SortField(TOTAL_DIFFICULTY.fieldName, SortField.Type.STRING, true))
       )
-
-      for (hit in topDocs.scoreDocs) {
-        val doc = searcher.doc(hit.doc, setOf("_id"))
-        val bytes = doc.getBinaryValue("_id")
+      val doc = searcher.doc(topDocs.scoreDocs.elementAt(0).doc, setOf("_id"))
```

Comment:
What if scoreDocs is empty? Would be good to check the element exists, and return null early otherwise.

Reply:
### Looking into it now